src/codeu/chat/server/Server.java

```diff
@@ -53,23 +54,31 @@
   private final Timeline timeline = new Timeline();
 
   private final Map<Integer, Command> commands = new HashMap<>();
-
   private final Uuid id;
   private final Secret secret;
-
+  private ServerInfo info = null;
   private final Model model = new Model();
   private final View view = new View(model);
   private final Controller controller;
-
   private final Relay relay;
   private Uuid lastSeen = Uuid.NULL;
+  // This string should be changed whenever the initial version value for the server
+  // needs to be changed
+  private String version = "1.0.0";
 
   public Server(final Uuid id, final Secret secret, final Relay relay) {
 
     this.id = id;
     this.secret = secret;
     this.controller = new Controller(id, model);
     this.relay = relay;
+    // This try catch block initializes the server version based on the version
+    // set in object 'version'
+    try {
+      info = new ServerInfo(Uuid.parse(version));
+    } catch (IOException e) {
+      System.out.println("Invalid input");
```

Comment:
This exception shouldn't ever happen in real life, right? I'd add info the the `println()` saying to check the version number, since that's the only way to cause this exception.